x11-terms/alacritty/alacritty-9999.ebuild

```diff
@@ -7,10 +7,8 @@ CRATES="
 "
 
 MY_PV="${PV//_rc/-rc}"
-# https://bugs.gentoo.org/725962
-PYTHON_COMPAT=( python3_{9..12} )
```

Comment:
I think it's not the case for quite a while since we don't have a crate that was bringing python in (xcb).